app/src/main/kotlin/com/wire/android/ui/home/newconversation/search/SearchPeopleScreen.kt

```diff
@@ -133,6 +140,7 @@ private fun LazyListScope.internalSearchResults(
     showAllItems: Boolean,
     onShowAllButtonClicked: () -> Unit,
     onOpenUserProfile: (Contact) -> Unit,
+    invalidQueryMessage: String
```

Comment:
if we assign the message inside the viewmodel this way : 

how about instead of making two Failure hierarchy types we do something like this :

data class Failure(val message : String? = null) : SearchResultState()

then inside the VIewModel we can assign the message depending on the reason of the failure like :

SearchResultState.Failure("Not found") or
SearchResultState.Failure("No results")

you could remove this parameter